src/index.js

```diff
@@ -1,36 +1,79 @@
 import './style.css';
 
-const toDoItems = [
-  {
-    info: 'Wash the dishes',
-    completed: true,
-    index: 1,
-  },
-  {
-    info: 'Set Appointment for Tomorrow',
-    completed: true,
-    index: 2,
-  },
-  {
-    info: 'Work on  my Thesis',
+const form = document.querySelector('form');
+const input = document.querySelector('.input');
+const todoContainer = document.querySelector('.todoContainer');
+
+let todo;
+let todos = JSON.parse(localStorage.getItem('todos')) || [];
+const store = () => {
+  todo = {
+    Description: input.value,
+    id: todos.length,
     completed: false,
-    index: 3,
-  },
-];
-
-toDoItems.forEach((val) => {
-  const itemsContainer = document.querySelector('.list-container');
-  const divElement = document.createElement('div');
-  divElement.id = val.index;
-  divElement.className = 'list-item';
-  divElement.innerHTML = `
-  <div class = "info">
-      <div class ="inner-info">
-        <input type="checkbox">
-        <span>${val.info}</span>
-        <i class="fa-solid fa-ellipsis-vertical" ></i>
-      </div>
-  </div>
-  `;
-  itemsContainer.appendChild(divElement);
+  };
+  todos.push(todo);
+  localStorage.setItem('todos', JSON.stringify(todos));
+};
+
+const clear = () => {
+  input.value = '';
+};
+
+const removeBook = (id) => {
+  todos = todos.filter((books) => books.id !== id);
+  todos.forEach((todo, id) => {
+    todo.id = id;
+  });
+  localStorage.setItem('todos', JSON.stringify(todos));
+};
+
+const addTask = (todo) => {
+  const ul = document.createElement('div');
+  const checkBox = document.createElement('input');
+  checkBox.type = 'checkbox';
+  checkBox.classList.add('checkBox');
+  const newInp = document.createElement('input');
+  newInp.type = 'text';
+  newInp.classList.add('newInput');
+  newInp.value = todo.Description;
+  const icon = document.createElement('i');
+  icon.classList.add('fa-solid');
+  icon.classList.add('fa-ellipsis-vertical');
+  icon.classList.add('dots');
+
+  ul.append(checkBox, newInp, icon);
+  todoContainer.append(ul);
+  icon.addEventListener('click', () => {
+    icon.parentElement.remove();
+    removeBook(todo.id);
+  });
+};
+todos.forEach(addTask);
+
+const editTodoList = () => {
+  const editInput = document.querySelectorAll('.newInput');
+  editInput.forEach((edits, indexy) => {
+    edits.addEventListener('change', () => {
+      todos.forEach((todo, index) => {
+        if (indexy === index) {
+          todo.Description = edits.value;
+          localStorage.setItem('todos', JSON.stringify(todos));
+        }
+      });
+    });
+  });
+};
+editTodoList();
```

Comment:
- [ ] You have edit task description functionality :clap: but you but it is not changing anything in the localStorage while after editing any task :clap: Kindly implement the functionality properly for editing the task :+1: